src/control.rs

```diff
@@ -64,7 +64,7 @@ impl<'l, W: Write> ControlArchiveBuilder<'l, W> {
     /// should be inserted.
     fn generate_scripts(&mut self, option: &Config) -> CDResult<()> {
         if let Some(ref maintainer_scripts_dir) = option.maintainer_scripts {
-            let maintainer_scripts_dir = option.package_manifest_dir.as_path().join(maintainer_scripts_dir);
+            let maintainer_scripts_dir = option.package_manifest_dir.as_path().join(maintainer_scripts_dir).canonicalize()?;
```

Comment:
When `maintainer_scripts` is set to a directory that does not exist this line caused the following output for me:

```
cargo-deb: I/O error: No such file or directory (os error 2)
  because: No such file or directory (os error 2)
```

I had no idea which part of the code was throwing it or which directory it was looking for.